docs/changelog/114739.yaml

```diff
@@ -0,0 +1,18 @@
+pr: 114739
+summary: Add a basic deprecation warning that the JSON format for non-detailed error responses is changing in v9
+area: Infra/REST API
+type: deprecation
+issues: [89387]
+deprecation:
+  title: The format of non-detailed error responses is changing in v9
+  area: REST API
+  details: |-
```

Comment:
Add `To keep using the existing format in v9, use the V8 REST API.` here as well?

Reply:
Good spot, changed the changelog